go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/submariner-io/admiral
 
-go 1.20
+go 1.21
+
+toolchain go1.21.5
```

Comment:
`go mod tidy` adds this automatically, but it’s not necessary and adds some constraints — see https://go.dev/doc/toolchain for details.

```suggestion
```